src/IceRpc.Slice/README.md

```diff
@@ -56,9 +59,10 @@ server.Listen();
 await CancelKeyPressed;
 await server.ShutdownAsync();
 
-// Class IceRpc.Slice.Service is an IceRPC dispatcher. This dispatch is implemented by
-// Chatbot and helper code provided by the IGreeterService interface generated by the
-// Slice compiler.
+// IGreeterService is an interface generated by the Slice compiler.
+// The [SliceService] attribute instructs the Slice Service source generator (provided by
+// IceRpc.Slice.Generators.dll) to implement IDispatcher by directing "greet" requests to
```

Comment:
```suggestion
// IceRpc.Slice.Generators.dll) to implement IDispatcher by directing "Greet" requests to
```

Like in the operation name

Reply:
The operation name is greet in lower-case.